spec/shr_allergy.txt

```diff
@@ -0,0 +1,128 @@
+		Value:			CodeableConcept from http://hl7.org/fhir/ValueSet/allergy-intolerance-type
+
+/*
+		Element:		ClinicalStatus
+		Concept:		TBD
+		Description:	"A flag indicating whether the condition is active or inactive, recurring, in remission, or resolved (as of the last update of the Condition)."
+		Value:			CodeableConcept from http://hl7.org/fhir/ValueSet/allergy-clinical-status
+
+		Element:		AllergyVerificationStatus  
+		Concept:		TBD
+		Description:	"Whether an assessment has been confirmed by testing or observation."
+		Value:			CodeableConcept from http://hl7.org/fhir/ValueSet/allergy-verification-status
+*/
+		Element:		MostRecentOccurrenceTime
+		Concept:		TBD
+		Description:	"The time of the last or latest of a series of events."
+		Value:			dateTime
+
+		Element:		AllergyIntoleranceReaction
+		Concept:		TBD
+		Description:	"Observed reaction event associated with the allergy or intolerance, expressed as a condition."
+		0..1			SubstanceCode
+		1..*			Manifestation
+		0..1			BeginDateTime   // from core
+		0..1			Severity from http://hl7.org/fhir/ValueSet/reaction-event-severity
+		0..1			RouteIntoBody
+		0..1			CommentOrDescription
+
+				Element:	SubstanceCode
+				Concept:	TBD
+				Description: "The substance triggering a particular allergy or intolerance reaction event. The substance may be the same as the substance cited in the AllergyIntolerance, but should be a specific substance rather than a class of substances, if known. For example, if the allergy intolerance FindingResult substance is 'shellfish' the substance code for the reaction might be 'oyster'."
+				Value:	    CodeableConcept from http://hl7.org/fhir/us/core/ValueSet-us-core-substance.html if covered
+
+				Element:	Manifestation
+				Concept:	TBD
+				Description:	"The clinical symptoms/signs associated with the reaction event."
+				Value:		CodeableConcept could be from http://hl7.org/fhir/ValueSet/clinical-findings
+
+
+
+
+
+
+
+/* Probably the most consistent way to define the absence (no known) allergy is to use an absence assertion, but this is not the way US Core has decided to go. Comments here are for reference only.
+	
+EntryElement:	AllergyIntoleranceAbsent
+Based on:		AbsenceAssertion
+Description:	"Used to record that a particular substance or class of substances does not pose a known elevated risk to the subject. The certainly of absence is always 'no known' because definitively ruling out an allergy intolerance is not possible."
+				FindingTopicCode is SCT#473010000 "Hypersensitivity condition (disorder)"  // We don't use a code that represents the absence of the condition and whose FindingResult would therefore be true or false (rather than present/absent)
+1..1			AllergenIrritant
+0..*			SubstanceCategory
+1..1			Certainty is SCT#443508001 "No history of clinical finding in subject (situation)" // no known
+
+
+EntryElement:	NoKnownDrugAllergy
+Based on:		AllergyIntoleranceAbsent
+
+TBD				AllergenIrritant is "Any drug"
```

Comment:
This is strange. I didn't know cardinality could be TBD? Did you mean to zero it out?

Reply:
This is in the middle of a comment block